unity-renderer/Assets/DCLServices/PlacesAPIService/PlacesAPIClient.cs

```diff
@@ -0,0 +1,120 @@
+﻿using Cysharp.Threading.Tasks;
+using DCL;
+using DCL.Helpers;
+using DCLServices.Lambdas;
+using MainScripts.DCL.Controllers.HotScenes;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using UnityEngine;
+using UnityEngine.Networking;
+
+namespace DCLServices.PlacesAPIService
+{
+    public interface IPlacesAPIClient
+    {
+        UniTask<IHotScenesController.PlacesAPIResponse> GetMostActivePlaces(int pageNumber, int pageSize, CancellationToken ct);
+        UniTask<IHotScenesController.PlaceInfo> GetPlace(Vector2Int coords, CancellationToken ct);
+
+        UniTask<IHotScenesController.PlaceInfo> GetPlace(string placeUUID, CancellationToken ct);
```

Comment:
Can you add `UUID` to the team-shared dictionary via Rider?

Reply:
<img width="769" alt="image" src="https://github.com/decentraland/unity-renderer/assets/118179774/d3c46cf3-cabe-4e41-af8c-29ad918b7be2">

Reply:
Neat! done